Summaryfield mnrva 483

FarajiA opened the pull request:
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-483

 ## Description:
In the case of a monitor not having a name associated as a fallback we are concatenating:
Type of plugin monitor
First summary field
Last five characters of the monitor ID 

 ## Testing:
`npm run test`

 ## Screenshots:
(if applicable)

FarajiA commented:
One more test to add - standby, I'm unsure why there's so many files it shows edited but I only made edits to 5 files.

pras7430 commented on src/app/_features/monitors/mon.utils.ts:
```diff
@@ -187,5 +188,8 @@ export class MonotorUtil {
         return vals.length > 0 ? vals : null;
     }
 
+    static formatSummaryField(monitor: Monitor) {
```
Can we consider "type-monitorid last five char" in case of summary does not have any props?
 
below is a concatenated string if summary props don't exist.
"cpu-undefined-13ff7"

FarajiA replied:
Yup, good catch. Looks like your update does the job.

FarajiA commented:
Added fallback for Monitor name details _monitor-details.page_
Added fallback for Monitor name breadcrumbs _monitor-details.page_
Added fallback for monitor list resource details page _monitor-list.component_
